tests/Behat/Content/Contact/GeneralContactContext.php

```diff
@@ -45,10 +46,14 @@ public function alterGeneralContactFields(BeforeParseEntityFieldsScope $scope):
     ];
 
     foreach ($scope->getFields() as $key => $value) {
+      $field_config = NULL;
```

Comment:
I wonder if instead of duplicating this piece of code everywhere it maybe should be part of the getreferencefield method. I leave this comment for the next reviewer to give their opinion

Reply:
Yes indeed it can be also done inside the method, but in this case, anyway, we have to transfer information about entity_type/bundle/field_name to the method. I stayed for now with an approach for passing ready-to-use information in getReferenceField() for keeping the method simple as possible.
But it is discussable.